mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -49,15 +49,13 @@ private[feature] trait ChiSqSelectorParams extends Params
    *
    * @group param
    */
-  @Since("1.6.0")
```

Comment:
Yeah, theoretically we should do that, but I'm not very confidence whether this change is appropriate. If we meet an agreement on how to deal with this issue, we can address other places in this PR or follow-up work. cc @jkbradley